cmd/account/servicequotas/describe.go

```diff
@@ -0,0 +1,106 @@
+package servicequotas
+
+import (
+	"fmt"
+
+	//"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/servicequotas"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	k8spkg "github.com/openshift/osd-utils-cli/pkg/k8s"
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+)
+
+// newCmdDescribe implements servicequotas describe
+func newCmdDescribe(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	ops := newDescribeOptions(streams, flags)
+	describeCmd := &cobra.Command{
+		Use:               "describe",
+		Short:             "Describe AWS service-quotas",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.complete(cmd))
+			cmdutil.CheckErr(ops.run())
+		},
+	}
+
+	ops.k8sclusterresourcefactory.AttachCobraCliFlags(describeCmd)
+
+	describeCmd.Flags().StringVarP(&ops.queryQuotaCode, "query-quota-code", "q", "", "Query for QuotaCode")
+	describeCmd.Flags().BoolVarP(&ops.verbose, "verbose", "v", false, "Verbose output")
+
+	return describeCmd
+}
+
+// describeOptions defines the struct for running list account command
+type describeOptions struct {
+	k8sclusterresourcefactory k8spkg.ClusterResourceFactoryOptions
+
+	queryQuotaCode string
+
+	verbose bool
+
+	genericclioptions.IOStreams
+}
+
+func newDescribeOptions(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *describeOptions {
+	return &describeOptions{
+		k8sclusterresourcefactory: k8spkg.ClusterResourceFactoryOptions{
+			Flags: flags,
+		},
+		IOStreams: streams,
+	}
+}
+
+func (o *describeOptions) complete(cmd *cobra.Command) error {
+	k8svalid, err := o.k8sclusterresourcefactory.ValidateIdentifiers()
+	if !k8svalid {
+		if err != nil {
+			return err
+		}
+	}
+
+	awsvalid, err := o.k8sclusterresourcefactory.Awscloudfactory.ValidateIdentifiers()
+	if !awsvalid {
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *describeOptions) run() error {
+	awsClient, err := o.k8sclusterresourcefactory.GetCloudProvider(o.verbose)
+	if err != nil {
+		return err
+	}
+
+	var serviceCode string = "ec2"
+	var searchMaxResults int64 = 100
+
+	servicequotas, err := awsprovider.Client.ListServiceQuotas(awsClient, &servicequotas.ListServiceQuotasInput{
+		ServiceCode: &serviceCode,
+		MaxResults:  &searchMaxResults,
+	})
+	if err != nil {
+		return err
+	}
+
+	if o.queryQuotaCode == "" {
+		fmt.Println(servicequotas.Quotas)
+	} else {
+		for _, quota := range servicequotas.Quotas {
+			if *quota.QuotaCode == o.queryQuotaCode {
```

Comment:
We should be more verbose here when the specified quota is not found, outputting a msg for example.

Reply:
added error return when not found